holidays/calendars.py

```diff
@@ -942,3 +945,1420 @@ def ok_phansa_date(self, year: int) -> Optional[date]:
         else:
             delta_days = 309
         return start_date + td(days=delta_days)
+
+
+class _IslamicHolidays:
+    DATES = {
+        # Eid al-Fitr
+        "eid-al-fitr": {
```

Comment:
I liked non-nested version better. Also the names are quite self-explanatory.
If you migrated them just for dict attribute access then I suggest using getattr instead.

Reply:
Changed to separate holiday arrays.